src/anomalib/data/utils/augmenter.py

```diff
@@ -111,7 +111,7 @@ def generate_perturbation(
 
         # Load anomaly source image
         if anomaly_source_path:
-            anomaly_source_img = cv2.imread(anomaly_source_path)
+            anomaly_source_img = cv2.imread(str(anomaly_source_path))
```

Comment:
Should we be converting the image to RGB? Would it affect the distribution of the anomalous image?

Reply:
I don't think it would make a lot of difference since the source images serve as random noise, but I agree that technically we should read the images as RGB. Let's address this separately.

Reply:
maybe on another pr, would it be possible to use PIL Image for this as well?